### ISSUE TITLE
Chrome support?

### ISSUE THREAD

**Issue by sammacbeth** [sammacbeth/dat-fox#14]

**Comment by soyuka** [comment]
Hi! 

I've looked into cross-browser webextensions as well, I've seen that the [webextension-toolbox](https://github.com/HaNdTriX/webextension-toolbox) is using polyfils for chrome. He's not using `web-ext` though, but maybe some ideas could be borrowed there.

I've also combined the docs for the native messaging paths (being multi-platform (currently only chrome, chromium and ff) and multi-os) in [this json file](https://github.com/soyuka/native-installer/blob/master/paths.json).

Cheers

**Comment by HughIsaacs2** [comment]
Hey, some time ago, I've built a Chrome extension just like this

https://github.com/hughisaacs2/datpart

We should collaborate in some form.

**Comment by sammacbeth** [comment]
@HughIsaacs2 yeah that would be cool. Most of the Firefox API I use can be polyfilled with [webextension-polyfill](https://github.com/mozilla/webextension-polyfill). I'm not sure how easy it is to get the proxy behaviour on Firefox working on Chrome.

**Comment by HughIsaacs2** [comment]
@sammacbeth the Proxy part was no problem, I figured that all out not long ago.

The issue was everything else lol.

What I was thinking is we should figure a way to keep compatibility with the app serving up the sites to the extensions, mine uses a TLD to ensure that the extension doesn't interfere with any regular web sites, so we're incompatible on the server part.

The rest shouldn't be too difficult since everyone seems to be tackling the same problems at the same time. You, the Bunsen guys, etc...
